### ISSUE TITLE
Bot grab fails

### ISSUE THREAD

**Issue by EranSch** [hect1c/skitty_v2#22]
With a working permissions system back in place, the next step IMO is to get skitty grabbing songs. This took some tweaking to get operational but the function now works all the way to the point of PlugAPI making a REST call to plug. This fails with the following error:

```
26 Sep 10:28:50 [ERROR]    [PlugAPI]  [REST Error] [SyntaxError: Unexpected token <]
26 Sep 10:28:50 [ERROR]    [PlugAPI]  [REST Error] [SyntaxError: Unexpected token <]
26 Sep 10:28:50 [ERROR]    [PlugAPI]  [REST Error] [SyntaxError: Unexpected token <]
```

After playing with the REST api a little bit, I'm guessing the referred to, unexpected, token `<` is the opening character to an error message probably saying something about a malformed request. Otherwise, the response would have been sweet, sweet JSON. 

TLDR; we need to fix this in PlugAPI before we can move on.

Note: The bot needs to have a playlist with a name that matches the [`defaultPlaylist`](https://github.com/hect1c/skitty_v2/blob/master/app/bot/index.js#L55) property defined in the bot's index.js file. (For now that name is 'snags')
